source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -11,6 +11,15 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+March 28, 2023
+
+|bw| 0.9.9
```

Comment:
Based on the discussion today at the daily the version will also be bumped to 0.9.10